.changeset/large-oranges-accept.md

```
@@ -0,0 +1,13 @@
+---
+'@wkovacs64/eslint-config': patch
+---
+
+Update various dependencies.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)